cypress.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+  e2e: {
+    setupNodeEvents(on, config) {
+      // implement node event listeners here
+    },
+    baseUrl: 'http://localhost:3001',
+    chromeWebSecurity: false,
```

Comment:
Is this strictly required? This is often used to bypass CORS browser policies

Reply:
The desired environment to do the tests should be identical or nearly-identical to the reality, so we should avoid to change the environment if possible to make the tests reliable